code/modules/food_and_drinks/food/foods/misc_food.dm

```diff
@@ -63,17 +63,113 @@
 //		Salads		//
 //////////////////////
 
-/obj/item/reagent_containers/food/snacks/aesirsalad
+/obj/item/reagent_containers/food/snacks/salad // Abstract object used for inheritance. Should never spawn. Needed to not break icons.
```

Comment:
If I moved it up wouldn't that mean every future salad would have a bowl for their trash item? How would I set it so that the melon bowl wouldn't have one?

Reply:
Yeah, monster meat needs to be changed someday in the future too. Ideally, a generic salad recipee would be good instead of having to move something non-generic to that path.
If you want to set something to not have trash item, you just do `trash = null`

Reply:
Alright I'll make a generic salad out of the unused sprite.

And did figure that out thanks to DG's help, wasn't sure if it was acceptable to just use null like that; but I did, and it's fixed!